tests/schemas/valid/test_string_transforms.json

```diff
@@ -36,15 +36,15 @@
       "groups": [
         {
           "id": "group1",
-          "title": "",
+          "title": "Title",
           "blocks": [
             {
               "id": "block0",
               "type": "Question",
               "question": {
                 "id": "question0",
-                "title": "",
-                "description": "",
+                "title": "Title",
+                "description": "Title",
```

Comment:
This one comment makes it look like I have checked them all, which of course I have

Reply:
Removed it now